images/bpftool/Dockerfile

```diff
@@ -1,40 +1,39 @@
-# syntax=docker/dockerfile:1.1-experimental@sha256:de85b2f3a3e8a2f7fe48e8e84a65f6fdd5cd5183afa6412fff9caa6871649c44
+# syntax=docker/dockerfile:1.2
 
 # Copyright 2020 Authors of Cilium
```

Comment:
Since you're bumping copyright years, you forgot one here. :wink:

Reply:
👍 Updated 

Note: also removed the `syntax=docker/dockerfile` line as it was previously only needed to rely on not-yet-stable features from the `-experimental` release, but those feature have now long been stable so we don't need this (and this is consistent with [dockerfiles](https://github.com/search?q=repo%3Acilium%2Fcilium%20path%3A%2F%5Eimages%5C%2F%2F%20syntax%3Ddocker&type=code) in the cilium/cilium repo)